contrib/test_jenkins.sh

```diff
@@ -659,6 +659,25 @@ check_make_distcheck() {
 	fi
 }
 
+check_config_h() {
+	echo 1..1 > check_config_h.tap
+
+	srcdir=$PWD/../src
+
+	# Check if all .c files include config.h
+	echo "==== Checking for config.h files in directory $srcdir ===="
+
+	missing=`find $srcdir -name \*.c |xargs grep -Le '#\s*include\s\+"config.h"'`
+	
+	if [ `echo $missing|wc -w` -eq 0 ];
+	then
+		echo echo "ok 1 - build successful " >> check_config_h.tap
+	else
+		echo Error: missing include config.h in files: $missing;
```

Comment:
plz remove `;` and better wrap the string with `""`

Reply:
removed ```;``` and wrapped to ```""```